run.py

```diff
@@ -509,6 +536,17 @@ def _synthesis_morphing(
         """
         engine = get_engine(core_version)
 
+        is_permitted = is_synthesis_morphing_permitted(
+            engine, base_speaker, target_speaker
+        )
+        if is_permitted is None:
+            raise HTTPException(status_code=404, detail="該当する話者が見つかりません")
+        if not is_permitted:
+            raise HTTPException(
+                status_code=400,
+                detail="指定された話者ペアでのモーフィング機能は禁止されています",
```

Comment:
すみません、ちょっと製品判断なのですが、ここ「できません」にしても良いでしょうか 🙇‍♂️ 
（API的には「禁止されています」のが正しいと思いつつ。。）
```suggestion
                detail="指定された話者ペアでのモーフィングはできません",
```